contents/stats.md

```diff
@@ -0,0 +1,445 @@
+# Statistics {#sec:stats}
+
+This content was created in order to awaken the reader to the importance of statistics for science and knowledge generation.
+Our idea is to present the concepts in the way we would like to have been provided when students about to be initiated into science.
+Our approach is to simplify the concepts as much as possible without losing their essence.
+**In this chapter, we will cover**:
+
+* what is **statistics**
+* the difference between **descriptive statistics** and **inferential statistics**
+* measures of **central tendencies**
+* measures of **dispersion**
+* measures of **dependence**
+* **probability distributions**
+* statistical **visualization**
+
+Statistics is important because it is a **tool to make sense of data**.
+With the abundant availability of data, we are often overwhelmed by numbers.
+Statistics offers a way to comprehend, summarize and infer information from data.
+We believe that every data scientist should have a basic understanding of statistics and how to perform simple statistical operations.
+
+We can divide statistics into two broad categories: **descriptive** and **inferential**.
+**Descriptive statistics summarizes and quantifies the characteristics of a given observed data**.
+Common metrics are: mean, median, mode, standard deviation, variance, correlation, percentiles.
```

Comment:
```suggestion
Common descriptives are: mean, median, mode, standard deviation, variance, correlation and percentile.
```

Reply:
They are still metrics. We can use them for inference. I don't think they are descriptives. Maybe common "descriptive metrics"?

My point is that we can have all of those metrics in a parameter estimation setting for example.